### ISSUE TITLE
Substitute data for modelled IPHC map

### ISSUE THREAD

**Issue by seananderson** [pbs-assess/gfsynopsis#49]
To convert longitude and latitude to UTMs

gfplot:::ll2utm()

Make the data match this format. The important columns are X, Y, combined, survey. You can ignore the rest. This will be inserted as iphc_fits$pred_dat. The combined is the estimate.

I can help actually integrate it.

```
         X        Y akima_depth depth_scaled depth_scaled2     combined pos bin    survey
1 249.8263 6027.495       380.4    1.9480471     3.7948874 1.010944e-04  NA  NA IPHC FISS
2 231.5991 6028.491       460.9    2.2949877     5.2669684 6.134827e-05  NA  NA IPHC FISS
3 227.8670 6009.485        49.4   -1.7413082     3.0321544 5.964096e+02  NA  NA IPHC FISS
4 191.8953 6030.973       246.9    1.1668254     1.3614815 3.343166e-01  NA  NA IPHC FISS
5 210.3277 6029.695       221.3    0.9689821     0.9389263 6.071074e+01  NA  NA IPHC FISS
6 209.3769 6011.210       385.9    1.9739919     3.8966440 1.721172e-03  NA  NA IPHC FISS
```

**Comment by andrew-edwards** [comment]
Make a function gfplot::format_final_year_for_map, that takes Series C values (all hooks, full coast, as we have that) for the final year [if doesn't exist then give error, as will have to manually decide in a future year what to show, depending on what the survey did). Just need the lats and lons converted, catch rates as combined, and rest as NA's.

**Comment by andrew-edwards** [comment]
Do you know the name off hand for the function that uses the data (just for my help file in gfplot)?

**Comment by andrew-edwards** [comment]
May need a slightly different function anyway as it's better to plot the actual data as circles (catch per effective skate at each station). Ideally with empty circle for zero catch (will show up well). I was going to include the unusable stations but will just filter them out (actually only one for 2017).

**Comment by andrew-edwards** [comment]
Return a row of NA's if no data (like the time series)?

**Comment by andrew-edwards** [comment]
Have done a new function in the gfplot branch `iphc-update` ; don't want to merge it into master yet (I did some other commits that I need to fully test when deleting a cache).

Function is `format_final_year_for_map()` and returns the output as you described above, and a row of NA's (except "IPHC FISS") if no data. 

I've looked into where I think it gets plotted, but easier if we do it together tomorrow maybe.
